tests/bson/bug0325.phpt

```diff
@@ -0,0 +1,21 @@
+--TEST--
+Test for PHPC-325: Memory leak decoding buffers with multiple documents
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$bson1 = MongoDB\BSON\fromJSON('{"x": "y"}');
+$bson2 = MongoDB\BSON\fromJSON('{"a": "b"}');
+$value = MongoDB\BSON\toArray($bson1 . $bson2);
+
```

Comment:
Noted. I pulled this right from the ticket, but I did see you made those global functions for a reason :)